src/com.target.lorri.varlink

```diff
@@ -0,0 +1,53 @@
+# The interface `lorri daemon` exposes.
+interface com.target.lorri
+
+# WatchShell instructs the daemon to evaluate a Nix expression and re-evaluate
+# it when it or its dependencies change.
+method WatchShell(shell_nix: ShellNix) -> ()
+
+# ShellNix describes the Nix expression which evaluates to a development
+# environment.
+type ShellNix (
+  # The absolute path of a Nix file specifying the project environment.
+  path: string
```

Comment:
I wonder if we can somehow indicate to varlink that this must be converted in to a Path/PathBuf, to avoid conversions

Reply:
I don't think the Varlink code generation can be modified in that way without major hacks, but https://github.com/target/lorri/pull/253#discussion_r356688072 works pretty well!

Reply:
For reference, here is the list of Varlink-supported types: https://varlink.org/Interface-Definition